src/main/java/commands/removeCommand.java

```diff
@@ -0,0 +1,26 @@
+package commands;
+import storage.Storage;
```

Comment:
Perhaps adding a line between `package commands;` and `import storage.Storage;`? I found the same issue in several other places too!

Reply:
Agreed! Thank you!